crates/swift-bridge-cli/src/main.rs

```diff
@@ -1 +1,5 @@
-fn main() {}
+use swift_bridge_cli::app::{cli, handle_matches};
```

Comment:
The book chapter on Swift packages should mention the `swift-bridge` CLI command

i.e. with a little bit of text and an example like:

```
cargo install -f swift-bridge-cli
swift-bridge create-package ...
```

Similar to what we do with the API section of the `Swift Package` chapter